components/announcement.tsx

```diff
@@ -0,0 +1,18 @@
+interface AnnouncementProps {
+  children: JSX.Element | JSX.Element[]
+  variant: "info" | "success" | "warning"
+}
+
+export default function Announcement({ children, variant }: AnnouncementProps): JSX.Element {
+  let className = "lbh-page-announcement"
+
+  if (variant) {
+    className += ` ${className}--${variant}`
+  }
+
+  return (
+    <section className={className}>
+      {children}
```

Comment:
This doesn't appear to tally up with the design system, note the lack of `title` and `content` which I think would be required if we want to keep it as a `section` where a heading should be provided.
https://design-system.hackney.gov.uk/components/page-announcement

Reply:
At the moment I'm using it to wrap the table, we might need to move away from the page announcement component in the future.